mne/io/boxy/tests/test_boxy.py

```diff
@@ -0,0 +1,296 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziek@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+import os
+
+import numpy as np
+from numpy.testing import assert_allclose, assert_array_equal
+import scipy.io as spio
+
+import mne
+from mne.datasets.testing import data_path, requires_testing_data
+
+
+@requires_testing_data
+def test_boxy_load():
+    """Test reading BOXY files."""
+    # Determine to which decimal place we will compare.
+    thresh = 1e-10
+
+    # Load AC, DC, and Phase data.
+    boxy_file = os.path.join(data_path(download=False),
+                             'BOXY', 'boxy_0_40_recording',
+                             'boxy_0_40_notriggers_unparsed.txt')
+
+    boxy_data = mne.io.read_raw_boxy(boxy_file, verbose=True).load_data()
+
+    # Test sampling rate.
+    assert boxy_data.info['sfreq'] == 62.5
+
+    # Grab our different data types.
+    chans_dc = np.arange(0, 80) * 3 + 0
+    chans_ac = np.arange(0, 80) * 3 + 1
+    chans_ph = np.arange(0, 80) * 3 + 2
+
+    mne_dc = boxy_data.copy().pick(chans_dc)
```

Comment:
Now the returned types are correct rather than the arbitrary indexing above this can be simplifed to something like...
```suggestion
    mne_dc = boxy_data.copy().pick(picks='fnirs_fd_ac_amplitude')
```

Reply:
yup
changed to
```
   # Grab our different data types.
    mne_ph = boxy_data.copy().pick(picks='fnirs_fd_ph')
    mne_dc = boxy_data.copy().pick(picks='fnirs_fd_dc_amplitude')
    mne_ac = boxy_data.copy().pick(picks='fnirs_fd_ac_amplitude')
```